### DISCUSSION TITLE
Where is the model being saved?

### DISCUSSION THREAD

**Discussion by vahuja4** [jina-ai/finetuner#172] [Q&A] [answered]
I am not able to figure out where the saved model lives. Where does finetuner save the model?

**Comment by hanxiao** [comment, marked as answer]
you are right, there is none. 🤣  we just expose this API in `0.1.2`, please upgrade via `pip install --pre finetuner`.

After that `finetuner.fit` will return two objects: first is the tuned model, second is the summary data of the fitting procedure, where you can plot it and understand the fitting better. You can refer to the docs for more https://finetuner.jina.ai/basics/fit/#
